src/test/java/teammates/test/cases/storage/FeedbackResponseCommentsDbTest.java

```diff
@@ -323,7 +323,7 @@ public void testUpdateFeedbackResponseComment_singleFieldUpdate_shouldUpdateCorr
                         .build());
         actualComment = frcDb.getFeedbackResponseComment(typicalComment.getId());
         assertEquals(Instant.ofEpochMilli(1000), updatedComment.getLastEditedAt());
-        assertEquals(Instant.ofEpochMilli(1000), actualComment.getLastEditedAt());
+        assertNotEquals(Instant.ofEpochMilli(1000), actualComment.getLastEditedAt());
```

Comment:
I'm not sure why this should not be equal as well?

Reply:
When user didn't change the comment at all and submit the response, everything else of the comment is the same except for the lastUpdateTime, but in this case, the lastUpdateTime should't change because use didn't do any modification. Therefore, when this test tries to update the lastUpdateTime only, it will remain the same.